css/style.css

```diff
@@ -62,19 +61,38 @@ section {
   text-decoration: none;
 }
 
-nav ul {
+.menu {
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: #6070ffe0;
+  padding-top: 50px;
   display: none;
 }
```

Comment:
- [ ] It would be better to make the menu fixed on the whole screen so your page doesn't scroll vertically.
I would recommend changing the position to `fixed` and trying to stop the body from scrolling after opening the menu.